clarify: Fusion CLI and VS Code extension are free to use (#9050)
fishtown-analytics/docs.getdbt.com: website/snippets/_fusion_availability.md

:::tip Free to use

> The Fusion CLI and VS Code extension are free to download and use — no dbt platform subscription required. Fusion in the dbt platform is also available in private preview across **all plan tiers**, including the free Developer plan. [Get started](/guides/fusion) or [install the VS Code extension](/docs/about-dbt-extension).

Author: ```suggestion
The <Constant name="fusion"/> CLI and dbt VS Code extension are free to download and use &mdash; no <Constant name="dbt_platform"/> subscription required. <Constant name="fusion"/> in the<Constant name="dbt_platform"/> is also available in private preview across **all plan tiers**, including the free Developer plan. [Get started](/guides/fusion) or [install the VS Code extension](/docs/about-dbt-extension).
```

Try out the [<Constant name="fusion" /> quickstart](/guides/fusion) and check out the [<Constant name="fusion" /> migration guide](/docs/dbt-versions/core-upgrade/upgrading-to-fusion) to see how to migrate your project.
:::

> <Constant name="fusion_engine" /> powers dbt development everywhere — in the [<Constant name="dbt_platform" />](/docs/dbt-versions/upgrade-dbt-version-in-cloud#dbt-fusion-engine), [VS Code/Cursor/Windsurf](/docs/about-dbt-extension), and [locally](/docs/local/install-dbt?version=2#installation). <Constant name="fusion"/> in the <Constant name="dbt_platform"/> is available in private preview across all plan tiers. Contact your account team for access.

Reviewer: ```suggestion
<Constant name="fusion_engine" /> powers dbt development everywhere, including the [<Constant name="dbt_platform" />](/docs/dbt-versions/upgrade-dbt-version-in-cloud#dbt-fusion-engine), [VS Code/Cursor/Windsurf](/docs/about-dbt-extension), and [locally](/docs/local/install-dbt?version=2#installation). <Constant name="fusion"/> in the <Constant name="dbt_platform"/> is available in preview across all plan tiers for supported adapters. Contact your account team for access.
```

| **Self-hosted Fusion** | - [dbt VS Code extension](https://marketplace.visualstudio.com/items?itemName=dbtLabsInc.dbt)<br /><small>(VS Code/Cursor/Windsurf)</small> <br />- [Fusion CLI](/docs/local/install-dbt?version=2#get-started) | - <Constant name="dbt_platform" /> users<br />- dbt <Constant name="fusion"/> users<br />- Anyone getting started with dbt | - **VS Code extension:** Combines <Constant name="fusion_engine"/> performance with visual <Term id="lsp"/> features when developing locally.<br /><br />- **Fusion CLI:** Provides <Constant name="fusion"/> performance benefits (faster parsing, compilation, execution) but _does not_ include <Term id="lsp"/> features. |
| **Self-hosted dbt Core** | - [dbt Core CLI](/docs/local/install-dbt) | - <Constant name="core" /> users<br /> - Anyone getting started with dbt   | Uses the Python-based <Constant name="core" /> engine for traditional workflows. _Does not_ include <Term id="lsp"/> features. <br /><br /> To use the <Constant name="fusion"/> features locally, install [the VS Code extension](/docs/local/install-dbt?version=2#get-started) or [Fusion CLI](/docs/local/install-dbt?version=2#get-started). |
<br />

> - The <Constant name="fusion"/> CLI and dbt VS Code extension are free to download and use &mdash; you don't need a <Constant name="dbt_platform"/> project. <Constant name="fusion"/> in the <Constant name="dbt_platform"/> is also available in private preview across _all plan tiers_, including our free Developer plan. [Get started](/guides/fusion) or [install the VS Code extension](/docs/about-dbt-extension).

Reviewer: ```suggestion
- The local <Constant name="fusion"/> CLI and dbt VS Code extension are free to download and use &mdash; you don't need a <Constant name="dbt_platform"/> project. <Constant name="fusion"/> in the <Constant name="dbt_platform"/> is also available in private preview across _all plan tiers_, including our free Developer plan. [Get started](/guides/fusion) or [install the VS Code extension](/docs/about-dbt-extension).
```
